src/_gettsim/parameters/eink_st_abzuege.yaml

```diff
@@ -656,6 +656,75 @@ kinderbetreuungskosten_abz_maximum:
   2012-01-01:
     scalar: 4000
     reference: Art. 1 G. v. 01.11.2011 BGBl. I S.2131
+vorsorge_pauschale_rv_anteil:
+  name:
+    de: Anteil abgezogene Rentenversicherungsbeiträge
+    en: Share of deducted pension insurance contributions
+  description:
+    de: >-
+      Funktion für den Anteil der Rentenversicherungsbeiträge, die bei der
+      Lohnsteuerberechnung abgezogen werden
+    en: >-
+      Function for the share of pension insurance contributions deducted in the payroll
+      tax calculation
+  unit: share
+  reference: § 39 (4) EStG
+  note:
+    de: >-
+      Funktion erst ab 2005 vorhanden. Steigt zwischen 2005 und 2025 jährlich um 4
+      Prozentpunkte.
+    en: rises between 2005 and 2025 by 4 percentage points annually.
```

Comment:
The jump in 2023 that we included in #476 for `einführungsfaktor` does not matter here? If it does, just copy the changes from there with appropriate adjustment.

Reply:
they do matter in a similar vein. corrected and tested now